packages/curve-ui-kit/src/shared/api/figma-tokens/extract-figma-values.ts

```diff
@@ -0,0 +1,73 @@
+/**
+ * Extracts values from a Figma token object, excluding specified keys.
+ * @param obj - The Figma token object to extract values from.
+ * @param excludeKeys - An array of keys to exclude from extraction.
+ * @returns An array of extracted values.
+ */
+export function extractValues<T extends FigmaValueObject<unknown, string>>(
+  obj: T,
+  excludeKeys: string[] = [],
+): ExtractedValues<T, (typeof excludeKeys)[number]> {
+  return Object.entries(obj)
+    .filter(([key]) => !excludeKeys.includes(key))
+    .map(([, value]) => value) as ExtractedValues<T, (typeof excludeKeys)[number]>
```

Comment:
Aren't the functions the same? :thinking: 
```suggestion
  return extractPairs(obj, excludeKeys).map(([k, v])=>v)
```